gammapy/makers/tests/test_spectrum.py

```diff
@@ -217,13 +217,13 @@ def test_safe_mask_maker_dl3(spectrum_dataset_crab, observations_hess_dl3):
 
 @requires_data()
 def test_safe_mask_maker_dc1(spectrum_dataset_gc, observations_cta_dc1):
-    safe_mask_maker = SafeMaskMaker(methods=["edisp-bias", "aeff-max"])
```

Comment:
@adonath Its green because I removed the edisp-bias here

Reply:
That's fine be me as well. However we should maybe make a reminder issue to investigate at some point why it failed...